nmtwizard/preprocess/tokenizer.py

```diff
@@ -1,9 +1,12 @@
+# coding: utf-8
 """Tokenization utilities."""
 
 import os
 import shutil
 import pyonmttok
 
+joiner_marker = "￭"
```

Comment:
This is OK for now but the joiner can actually be changed in the config. I should add an attribute to the `tokenizer` object in a future version of the OpenNMT Tokenizer.

Reply:
Yes, because joiner is not exposed in OpenNMT Tokenizer, I didn't find a better way to do it. We should definitely change it.